<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QComboBox" name="comboBox_InterfaceList"/>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_FileList"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTableWidget" name="tableWidget_NetMapSrc"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="BtnAddSrcMap">
              <property name="text">
               <string>Add Source IP Mapping</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="BtnRemoveScrIPMap">
              <property name="text">
               <string>Remove Source IP Mapping</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableWidget" name="tableWidget_NetMapDst"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="BtnAddDstMap">
              <property name="text">
               <string>Add Destination IP Mapping</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="BtnRemoveDstIPMap">
              <property name="text">
               <string>Remove Destination IP Mapping</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="LabelSchedulerStatus">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Speed Factor:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_SpeedFactor"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Loop:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_LoopCount"/>
        </item>
        <item>
         <widget class="QPushButton" name="BtnPlay">
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BtnPause">
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BtnStop">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BtnRemove">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BtnRemoveAll">
          <property name="text">
           <string>RemoveAll</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Fix Speed (MBit per Sec):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_SpeedLimit"/>
        </item>
        <item>
         <widget class="QPushButton" name="BtnRegularPlay">
          <property name="text">
           <string>Regular Replay</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File_Menu"/>
    <addaction name="actionOpen_Folder_Menu"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Config_Menu"/>
    <addaction name="actionLoad_Config_Menu"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionScheduler_Menu"/>
    <addaction name="actionConvert_Pcap"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_File_ToolBar"/>
   <addaction name="actionOpen_Folder_ToolBar"/>
   <addaction name="actionSave_Config_ToolBar"/>
   <addaction name="actionLoad_Config_ToolBar"/>
   <addaction name="actionScheduler_ToolBar"/>
  </widget>
  <action name="actionOpen_File_ToolBar">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionOpen_Folder_Menu">
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="actionSave_Config_Menu">
   <property name="text">
    <string>Save Config</string>
   </property>
  </action>
  <action name="actionLoad_Config_Menu">
   <property name="text">
    <string>Load Config</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionScheduler_Menu">
   <property name="text">
    <string>Scheduler</string>
   </property>
  </action>
  <action name="actionConvert_Pcap">
   <property name="text">
    <string>Convert Pcap</string>
   </property>
  </action>
  <action name="actionOpen_File_Menu">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionOpen_Folder_ToolBar">
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="actionSave_Config_ToolBar">
   <property name="text">
    <string>Save Config</string>
   </property>
  </action>
  <action name="actionLoad_Config_ToolBar">
   <property name="text">
    <string>Load Config</string>
   </property>
  </action>
  <action name="actionScheduler_ToolBar">
   <property name="text">
    <string>Scheduler</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
